tests/unit/ceed/test_ceed.cpp

```diff
@@ -13,19 +13,36 @@
 #include "mfem.hpp"
```

Comment:
I would like to see at least one test with multiple integrators, ie mass + diffusion. The machinery here is complex enough that it's not so obvious multiple integrators will work out of the box (I've tried it and they do, but I think it's worth having a test).

Reply:
I added a Mass+Diffusion test.